src/ex_qdstepandsavetopo.jl

```diff
@@ -37,11 +57,13 @@ function derivsquad!(dudt, u, p, t)
     return nothing
 end
 
-function ex_qdstepandsave()
+using Serialization
+
+# function ex_qdstepandsave()
```

Comment:
I should note that I only commented out this function definition for interactive development — it's nice to be able to just include this file and have all these constants and intermediate values already defined and ready to go so I can easily introspect and time and profile intermediate function calls.